### ISSUE TITLE
Have I made a mistake

### ISSUE THREAD

**Issue by trevormichaelfisher** [lukeed/bee3d#82]
I bought this from Codecanyon assuming it was a Wordpress plugin. Wordpress refuses to upload it. Have I bought this by mistake?

**Comment by lukeed** [comment]
Yes, it's not a WordPress plugin. There's no mention of WP anywhere in the description.
